Dockerfile

```diff
@@ -3,5 +3,10 @@ FROM busybox:ubuntu-14.04
 RUN mkdir -p /opt
 ADD ./aws-operator /opt/aws-operator
 
+RUN mkdir -p /opt/resources
```

Comment:
No, we can still use it. The work dir is set to /opt so the path is the same when run from docker or as a binary.